cryptlex/cryptlex-enterprise/values.yaml

```diff
@@ -250,8 +247,13 @@ webApi:
   # Postmark
   postmark:
     accountToken: ""
-# SSO config
-sso:
-  google:
+# Google
+google:
```

Comment:
@mumin-khan please be carefuf in yaml, incorrect spacing changes the meaning of the property. google is wrongly spaced

Reply:
@adnan-kamili I don't think so.

Reply:
![image](https://github.com/user-attachments/assets/3754a341-37e6-4a6d-8c23-bfc402cbe43e)
Although I agree I have made a few of silly mistakes from the past few days.